store/doi.js

```diff
@@ -0,0 +1,39 @@
+import { action, observable } from 'mobx'
+
+import { Pages } from './helpers/pages'
+
+import apiRequest from 'api'
+import { NotFoundError } from 'api/errors'
+
+const { API_URL } = process.env
+
+class DOI {
+  @observable isExportDisabled = false
+
+  @observable doiCount = 0
+
+  @observable doiRecords = null
+
+  constructor(baseUrl) {
+    const doiUrl = `${baseUrl}/doi`
+    this.doiRecords = new Pages(doiUrl)
+    this.doiUrl = `${API_URL}${doiUrl}?accept=text/csv`
+
+    this.loadDOICount()
+  }
+
+  @action
+  loadDOICount = async () => {
+    try {
+      this.isExportDisabled = false
+      const { headers } = await apiRequest(this.doiUrl, { method: 'HEAD' })
```

Comment:
After extending Store

```suggestion
      const { headers } = await this.request(this.doiUrl, { method: 'HEAD' })
```